src/Enghouse/readme.md

```diff
@@ -0,0 +1,67 @@
+# Enghouse Interactive CCaaS Integration
+
+The **Enghouse CCaaS** integration enables integration between IBM Watson Assistant (WA) and Enghouse Interactive (EI) **CCaaS** tenants' live agents.
+
+## Overview
```

Comment:
Need to collapse these Overview sections together. Order should be Overview, Setting Up, Example, Demo. If you want to include the Demo as the first thing, you can put it under the title.